pkgs/development/libraries/science/math/libtorch/binary-hashes.nix

```diff
@@ -16,4 +16,21 @@ version : builtins.getAttr version {
       hash = "sha256-jPylK4j0V8SEQ8cZU+O22P7kQ28wanIB0GkBzRGyTj8=";
     };
   };
```

Comment:
Why are the old hashes not overwritten?

Reply:
Sorry for inadequate checks. I should remove the argument of version in binary-hashes.nix.